kroxylicious-multitenant/pom.xml

```diff
@@ -146,9 +149,16 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <configuration>
-                    <!-- Javadoc generation fails to resolve jackson databind.  KW doesn't understand why -->
-                    <!-- Turn off javadoc generation for this module to work-around -->
-                    <skip>true</skip>
+                    <!-- Javadoc generation fails to resolve jackson databind.  KW & SB don't understand why -->
+                    <!-- Databind is in test scope, and we are only generating javadocs for src/main -->
+                    <!-- so explicitly add the dependency, so we can build the javadoc -->
+                    <additionalDependencies>
+                        <additionalDependency>
+                            <groupId>com.fasterxml.jackson.core</groupId>
+                            <artifactId>jackson-databind</artifactId>
+                            <version>2.14.2</version>
+                        </additionalDependency>
+                    </additionalDependencies>
```

Comment:
This is rubbish for a few reasons:

1. AFICT we shouldn't need databind on the class path as we should only generate javadoc for `src/main`
2. `databind` is already on the test classpath 
3. The class that fails javadoc generation is in `target/generated-test-sources/` so it possibly implies that we have `target/generated-test-sources/` on the production classpath
4. `additionalDependecies` doesn't resolve via the `dependencyManagment` section so it misses the bom and thus requires an explicit version.